dbbackup/tests/utils.py

```diff
@@ -10,9 +15,22 @@
 
 GPG_PRIVATE_PATH = os.path.join(settings.BASE_DIR, 'tests/gpg/secring.gpg')
 GPG_PUBLIC_PATH = os.path.join(settings.BASE_DIR, 'tests/gpg/pubring.gpg')
+GPG_FINGERPRINT = '7438 8D4E 02AF C011 4E2F  1E79 F7D1 BBF0 1F63 FDE9'
 DEV_NULL = open(os.devnull, 'w')
```

Comment:
I did it but only in memory with a custom dict: HANDLED_FILES.
I was inspired by fake mail box in django's test.

> Le 4 août 2015 à 13:29, benjaoming <notifications@github.com> a écrit :
> 
> In dbbackup/tests/utils.py:
> 
> > @@ -10,9 +15,22 @@
> >  
> >  GPG_PRIVATE_PATH = os.path.join(settings.BASE_DIR, 'tests/gpg/secring.gpg')
> >  GPG_PUBLIC_PATH = os.path.join(settings.BASE_DIR, 'tests/gpg/pubring.gpg')
> > +GPG_FINGERPRINT = '7438 8D4E 02AF C011 4E2F  1E79 F7D1 BBF0 1F63 FDE9'
> >  DEV_NULL = open(os.devnull, 'w')
> In case you need to verify that data is written correctly and can be restored, you might need a persistent storage? Like tempfile.mkdtemp.
> 
> —
> Reply to this email directly or view it on GitHub.
>